postgres-appliance/Dockerfile

```diff
@@ -513,9 +515,19 @@ WORKDIR $PGHOME
 COPY motd /etc/
 COPY runit /etc/service/
 COPY pgq_ticker.ini $PGHOME/
+COPY scripts bootstrap /scripts/
 
 RUN sed -i "s|/var/lib/postgresql.*|$PGHOME:/bin/bash|" /etc/passwd \
-        && chown -R postgres:postgres $PGHOME $RW_DIR \
+        && if [ "$COMPRESS" != "true" ]; then \
+           # non-root prep
+           mkdir -p $PGHOME \
+           && sed -i "s|/var/lib/postgresql.*|$PGHOME:/bin/bash|" /etc/passwd \
+           # Set permissions
+           && chmod 775 $PGHOME \
+           && chmod 664 /etc/passwd \
+           && mkdir -p $PGHOME/rw_dir \
+           && ln -s $PGHOME/rw_dir $RW_DIR/postgresql; \
```

Comment:
I still don't get the purpose of `$PGHOME/rw_dir`.

Reply:
I thought it was better to get it out of /run. I'll get rid of it, it is purely cosmetic anyway.